UsageStatistics_Page.php

```diff
@@ -31,12 +31,12 @@ public function render() {
 			return;
 		}
 
-		if ( isset( $_REQUEST['view'] ) && $_REQUEST['view'] == 'db_requests' ) {
+		if ( isset( Util_Request::get_string( 'view' ) ) && Util_Request::get_string( 'view' ) == 'db_requests' ) {
```

Comment:
Use `! empty()`.  You may want to set a variable to avoid calling twice.

Reply:
Changed to ! empty and created a variable to avoid 2 function calls.